intelmq/lib/datatypes.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2021 Birger Schacht
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from enum import Enum
+import json
+
+
+class BotType(str, Enum):
+    COLLECTOR   = "Collector"
+    PARSER      = "Parser"
+    EXPERT      = "Expert"
+    OUTPUT      = "Output"
+
+    def toJson(self):
+        return self.value
```

Comment:
Doesn't that miss the quotes necessary for making it a JSON-string?

Reply:
`toJson` is used by the JSON dump method to serialize individual values. In the case of the `BotType` class the dump method does not know how to represent the class (i.e. when dumping a bots attributes). Given that `self.value` is a string, the JSON dump method know how to handle the rest

Reply:
Thanks for the explanation!